pyseir/cli.py

```diff
@@ -52,8 +51,6 @@ def _cache_global_datasets():
     global nyt_dataset, cds_dataset
```

Comment:
I think if we dig into the places that use this we will find that they are stale and can be removed. I just removed a call from ensemble. Let's land both changes as they are and check then. But I'm pretty sure these can be removed from the function calls in cli.py and we can remove them!

Reply:
oh good call!